commands.txt

```diff
@@ -0,0 +1,7 @@
+available commands:
```

Comment:
Is this documentation for the user? Did you intend to upload it or should it belong in a README.md?

Reply:
no this was just a note. had deleted it!